content/posts/gsod-developing-matplotlib-entry-paths/index.md

```diff
@@ -0,0 +1,61 @@
+
+# Conclusion
+
+This Season of Docs proposal began as a vision of ideals I hoped to share and work towards with an organization and has become a technical writing experience full of growth and camaraderie. I am pleased with the progress I had made and cannot thank the team enough for the leadership and mentorship they provided. It is fulfilling and rewarding to both appreciate and be appreciated within a team.
+
+In addition, the opportunity put together by the team at Google to foster collaboration among skilled contributors cannot be understated. Highlighting the accomplishments of these new teams raises the bar for the open source community. 
+
+# Details
+
+## Acknowledgements
+
+Special thanks to Emily Hsu, Joe McEwen, and Smriti Singh for their time and responses, fellow Matplotlib Season of Docs writer Bruno Beltran for his insight and guidance, and the Matplotlib development team mentors Tim, Tom, and Hannah for their patience, support, and approachability for helping a new technical writer like me with my own Getting Started.
+
+## External Links
+
+- [Getting Started GSoD Pull Request](https://github.com/matplotlib/matplotlib/pull/18873)
+- [Matplotlib User Survey](https://docs.google.com/forms/d/e/1FAIpQLSfPX13wXNOV5LM4OoHUYT3xtSZzVQ6I3ZA4cvz5P6DKuph4aw/viewform?usp=sf_link)
+- [User Survey Responses](https://docs.google.com/spreadsheets/d/1z_bAu7hG-IgtFkM5uPezkUHQvi6gsWKxoDnh0Hz1K5U/edit?usp=sharing)
+- [User Survey Open Questions](https://docs.google.com/spreadsheets/d/15EzVNmWVn2SjCUBc-Kt5Y0_entLgvWRMRYy8syt_-Xg/edit?usp=sharing)
+- [HackMD GSoD Meeting Agenda](https://hackmd.io/cSNb2JhrSo26zJGag3bvLg)
+
+## About Me
+
+My name is [Jerome Villegas](https://www.linkedin.com/in/jeromefuertevillegas/) and I'm a technical writer based in Seattle. I've been in education and education-adjacent fields for several years before transitioning to the industry of technical communication. My career has taken me to Taiwan to teach English and work in publishing, then to New York City to work in higher education, and back to Seattle where I worked at a private school. 
+
+Since leaving my job, I've taken to supporting my family while studying technical writing at the University of Washington and supplementing the knowledge with learning programing on the side. Along with a former classmate, the two of us have worked with the UX writing community in the Pacific Northwest. We host interview sessions, moderate sessions at conferences, and generate content analyzing trends and patterns in UX/tech writing.
+
+In telling people what I've got going on in my life, I currently freelance and am open to projects and positions. You can find work I've done at my [personal site](https://jeromefvillegas.wordpress.com) and see what we're up to at [shift J](https://teamshiftj.wordpress.com). Thanks for reading!
```

Comment:
This blog is not for self-promotion as a consultant. You can leave the link to your website, but please remote that you are open to projects and positions.

Reply:
Sorry! I told him he could write that since he was working for matplotlib as part of GSOD. 
I think it's okay for folks who are hired by matplotlib to do work for matplotlib to self-promote. Am open to debate though

Reply:
I think we should be consistent regarding self-promotion. Perhaps a rewording would meet us half-way, saying something like 
_If you want to learn more about my current and future freelance projects, please visit my [personal site](https://jeromefvillegas.wordpress.com) and the [shift J website](https://teamshiftj.wordpress.com)_

Reply:
I've changed it to be less of a plug: "In telling people what I've got going on in my life, you can find what I've done..."

It's no problem to remove the "open to projects and positions" bit to be consistent with expectation! Thanks!